Don't install plugins that are already installed by sot-pattern-generator-v3

--- src/CMakeLists.txt.orig	2018-08-17 15:19:12.000000000 +0200
+++ src/CMakeLists.txt	2018-12-13 11:07:17.868523867 +0100
@@ -106,7 +106,7 @@
   PKG_CONFIG_USE_DEPENDENCY(${LIBRARY_NAME} pinocchio)
   PKG_CONFIG_USE_DEPENDENCY(${LIBRARY_NAME} sot-core)
 
-  INSTALL(TARGETS ${LIBRARY_NAME} DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/plugin)
+  #INSTALL(TARGETS ${LIBRARY_NAME} DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/plugin)
 
   # --- PY ---
   IF(BUILD_PYTHON_INTERFACE)
